init.sh

```diff
@@ -11,26 +11,35 @@ set -e
 # - the supported test generation tools
 # - the supported code coverage tools (TODO)
 ################################################################################
+#
+# Utility exit function
+#
+# Print an error message and terminate the script.
+# Takes two command-line arguments, a custom error message and an exit code.
```

Comment:
These are not command-line arguments.

Reply:
Why accept an error code when we just use `1` everywhere?

Reply:
> Why accept an error code when we just use `1` everywhere?

It's a choice. We don't need this. I'll just remove that.